### PR TITLE
Move backdrop transparency method to component

### PR DESCRIPTION
**Pull request by thornbill**
**Changes**
* Moves the method for updating the backdrop transparency to the backdrop component.
* Deprecates the `setTransparency` method in the app router until it can be removed from JMP.

**Issues**
N/A

### CODE REVIEW

File: src/utils/dashboard.js [hunk 1]
```diff
@@ -234,7 +235,8 @@ const Dashboard = {
     datetime,
     DirectoryBrowser,
     dialogHelper,
-    itemIdentifier
+    itemIdentifier,
+    setBackdropTransparency
```
Comment (thornbill) [inline src/utils/dashboard.js:239]: I'm not a fan of the `Dashboard` global as a whole, but this seems like the best place for now? I would be  curious what others think about this one.

**Review by dmitrylyzo — APPROVED** [review]

**Comment by iwalton3** [comment]
It looks like the Dashboard component needs to be added here to be accessible from JMP plugins: https://github.com/jellyfin/jellyfin-web/blob/1e2f52a28edafeae6441d5e0393488e4c1c5cafb/src/components/pluginManager.js#L79

**Comment by thornbill** [comment]
@iwalton3 the Dashboard component should be available on the window/global scope, but I do think it makes sense to include it in the utilities we use to initialize the plugins... that probably deserves an issue so we don't forget it :sweat_smile:

**Comment by iwalton3** [comment]
@thornbill here is the follow-up for this: https://github.com/jellyfin/jellyfin-web/pull/4373

**Pull request merged**